Detections/ASimWebSession/SuspiciousUAHackTool.yaml

```diff
@@ -0,0 +1,88 @@
+id: 3f0c20d5-6228-48ef-92f3-9ff7822c1954
+name: Suspicious User-Agent Detected - Hacking Tool
```

Comment:
Name: A host is potentially running a hacking tool (ASIM Web Session schema)

Description:
This rule identifies a web request with a user agent header known to belong to a hacking tool. This indicates a hacking tool is used on the host.<br><br>This rule uses the Advanced SIEM Information Model (ASIM) and supports any web session source that complies with ASIM. To use this Analytics Rule, [deploy the Advanced SIEM information Model (ASIM)](https://aka.ms/DeployASIM\).

alertDisplayNameFormat: Host {{SrcIpAddr}} is potentially running a hacking tool

alertDescriptionFormat:
The host at address {{SrcIpAddr}} sent an HTTP request to the URL {{Url}} with the HTTP user agent header {{HttpUserAgent}}. This user agent is known to be used by a hacking tool and indicates suspicious activity on the host.

Entities: URL, Username if available.

Custom fields: User agent.